dev/tools/svg2dart/test_assets/empty_svg_2_100x50.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Empty SVG file -->
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
```

Comment:
no need to list the namespaces you don't care about, just have the default svg one

Reply:
Removed, though I just kept this file here to make sure it doesn't crash (originally it was crashing on this :) ).